docs/docs/cmd/spo/eventreceiver/eventreceiver-remove.md

```diff
@@ -0,0 +1,73 @@
+# spo eventreceiver remove
+
+Gets a specific event receiver attached to the web, site or list (if any of the list options are filled in) by receiver name of id and removes this one.
```

Comment:
Let us shorten the description to something like this: Removes event receivers for the specified web, site, or list.